sqlserver/tests/compose-ha/sql/aoag_primary.sql

```diff
@@ -1,3 +1,7 @@
+-- Backup master
```

Comment:
This step is needed for both `pyXX-default` and `pyXX-ha` because the integration test expects that master has a backup value.